cloudformation/media-atom-maker.json

```diff
@@ -162,6 +162,11 @@
                                     "Effect": "Allow",
                                     "Action": ["sts:AssumeRole"],
                                     "Resource": { "Ref": "ContentAtomArn" }
+                                },
+                                {
+                                    "Effect": "Allow",
```

Comment:
You could merge this into the other sts:AssumeRole directive by adding this as just another resource in the list of resources. You might consider converting this to yaml in a future PR so you could add comments to describe any pieces that aren't immediately obvious.

Great to keep it tightly scoped like this.